components/admin_console/license_settings/license_settings.jsx

```diff
@@ -99,7 +99,13 @@ export default class LicenseSettings extends React.Component {
         let licenseType;
         let licenseKey;
 
-        const issued = <React.Fragment><FormattedDate value={new Date(parseInt(license.IssuedAt, 10))}/>{' '}{Utils.displayTime(parseInt(license.IssuedAt, 10), true)}</React.Fragment>;
+        const issued = (
+            <React.Fragment>
+                <FormattedDate value={new Date(parseInt(license.IssuedAt, 10))}/>
+                {' '}
+                <FormattedTime value={new Date(parseInt(license.IssuedAt, 10))}/>
```

Comment:
Looks like this should have props of:
```
    hour12={true}  // maybe nice if pass with "!this.props.militaryTime"
    hour='2-digit'
    minute='2-digit'
```

@hmhealey Thoughts?

Reply:
That doesn't seem necessary. I don't know exactly what the default values are, but at least for me, it's using the 12-hour clock like I'd expect
<img width="434" alt="Screen Shot 2019-11-05 at 5 04 20 PM" src="https://user-images.githubusercontent.com/3277310/68250201-66974600-ffee-11e9-92ce-0e170d98eec9.png">